src/coreclr/src/pal/src/exception/machexception.cpp

```diff
@@ -1439,6 +1439,22 @@ extern "C" void ActivationHandlerWrapper();
 extern "C" int ActivationHandlerReturnOffset;
 extern "C" unsigned int XmmYmmStateSupport();
 
+#ifdef HOST_ARM64
+bool isHardwareException(uint64_t esr)
```

Comment:
A nit - can you please change the casing of the function to pascal casing to match all other functions in this file?

Reply:
It would also be nice to make this function handle both arm64 and amd64 (pass in the ExceptionState as argument and check the __trapno for amd64 and __esr for arm64.